code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,421 @@
+		for(var/turf/T in landmarks[LANDMARK_TUTORIAL_FLOCKCONVERSION])
+			if(src.region.turf_in_region(T))
+				center = T
+				break
+		if (!center)
+			CRASH("Okay who removed the goddamn [LANDMARK_TUTORIAL_FLOCKCONVERSION] landmark")
+		src.fowner = M
+
+	Finish()
+		. = ..()
+		if (!.)
+			return FALSE
+		fowner.reset()
+		fowner.flock.perish(FALSE)
+		fowner.flock.enemies = list()
+		fowner.tutorial = null
+
+	proc/make_maptext(var/atom/target, var/msg)
+		msg = "<span class=\"ol vga c\" style=\"font-size:9pt\">[msg]</span>"
+		var/obj/dummy = new(get_turf(target))
+		var/image/chat_maptext/text = make_chat_maptext(dummy, msg, force = TRUE, time = INFINITY)
+		var/mob/actual_mob = src.fowner.abilityHolder.get_controlling_mob() //hunt for the client
+		text.show_to(actual_mob.client)
+
+	proc/portal_in(var/turf/location, var/type)
+		var/obj/portal/portal = new(location)
+		sleep(1 SECOND)
+		animate_portal_tele(portal)
+		playsound(portal.loc, "warp", 50, 1, 0.2, 1.2)
+		var/mob/jerk = new type(get_turf(portal))
+		step(jerk, SOUTH)
+		sleep(1 SECOND)
+		qdel(portal)
+		return jerk
+
+	ui_interact(mob/user, datum/tgui/ui)
+		ui = tgui_process.try_update_ui(user, src, ui)
+		if(!ui)
+			ui = new(user, src, "FlockStructures")
+			ui.open()
+
+	ui_static_data(mob/user)
+		var/list/structures = list()
+		var/list/structure_types = concrete_typesof(/obj/flock_structure)
+		for (var/type in structure_types)
+			var/obj/flock_structure/structure = type //funny type abuse moment (515 please save us)
+			if (!initial(structure.show_in_tutorial))
+				continue
+			structures += list(list(
+				"icon" = icon2base64(icon(initial(structure.icon), initial(structure.icon_state), frame = 1)),
+				"name" = initial(structure.flock_id),
+				"description" = initial(structure.tutorial_desc) || initial(structure.flock_desc), //default to the normal description
+				"cost" = initial(structure.resourcecost)
+			))
+		return list("structures" = structures)
+
+	ui_status(mob/user)
+		return istype(user, /mob/living/intangible/flock/flockmind) || tgui_admin_state.can_use_topic(src, user)
+
+/datum/tutorialStep/flock
+	name = "Flock tutorial step"
+	instructions = "If you see this, tell a coder!!!11"
+	var/static/image/marker = null
+	var/datum/tutorial_base/regional/flock/ftutorial = null
+
+	New()
+		..()
+		if (!marker)
+			marker = image('icons/effects/VR.dmi', "lightning_marker")
+			marker.filters= filter(type="outline", size=1)
+	SetUp()
+		. = ..()
+		src.ftutorial = src.tutorial
+
+	PerformAction(action, context)
+		return FALSE //fuck you, no action
+
+/datum/tutorialStep/flock/deploy
+	name = "Realizing"
+	instructions = "If at any point this tutorial glitches up and leaves you in a stuck state, use the emergency tutorial stop verb. <br /> Choose a suitable area to spawn your rift. In the real world you should try to choose an out of the way area with plenty of resources and delicious computers to eat. Here though, just deploy on the marked tile."
```

Comment:
```suggestion
	instructions = "If at any point this tutorial glitches up and leaves you in a stuck state, use the emergency tutorial stop verb, found by right clicking your Flockmind form. <br /> Choose a suitable area to spawn your rift. In the real world you should try to choose an out of the way area with plenty of resources and delicious computers to eat. Here though, just deploy on the marked tile."
```

Reply:
It isn't though? It's in the verbs tab.

Reply:
Okay can maybe clarify that then, it was just saying verb without saying where it was